dvc/command/metrics.py

```diff
@@ -12,33 +12,57 @@
 
 
 def _show_metrics(
-    metrics, all_branches=False, all_tags=False, all_commits=False
+    metrics,
+    all_branches=False,
+    all_tags=False,
+    all_commits=False,
+    precision=None,
```

Comment:
This one is not used right now, we need to add `--precision` to `metrics show` parser. See `--precision` at the very bottom of this file.